templates/admin/payments/payment.html

```diff
@@ -16,7 +16,7 @@ <h2>Payment {{ payment.id }}</h2>
 {% endif %}
 {% if payment.provider == 'banktransfer' and payment.state == 'inprogress' %}
 <a class="btn btn-primary" href="{{ url_for('.reset_expiry', payment_id=payment.id) }}">Reset expiry</a>
-<a class="btn btn-warning {% if payment.reminder_sent_at is not None -%} disabled {%- endif %}" href="{{ url_for('.send_reminder', payment_id=payment.id) }}">Remind</a>
+<a class="btn btn-warning {% if payment.reminder_sent_at is not none -%} disabled {%- endif %}" href="{{ url_for('.send_reminder', payment_id=payment.id) }}">Remind</a>
```

Comment:
`is none` was news to me, but yep, that [checks out](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-tests.none) - nice find.